<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>Fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_edition">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_affichage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Affichage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_fenetre">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Fenêtre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_calcul">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Calcul</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_couches">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Couches</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_message">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Athena</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_play">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/buttons/play-orange.png</normaloff>:/buttons/play-orange.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_pause">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/buttons/pause-orange.png</normaloff>:/buttons/pause-orange.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_stop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/buttons/stop-orange.png</normaloff>:/buttons/stop-orange.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_toolBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_donneesReseau">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Données du réseau</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/sidebar/mode_welcome_mask@2x.png</normaloff>:/sidebar/mode_welcome_mask@2x.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_dispoAnalytique">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Disponibilité analytique</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/mainToolBar/more_colors.png</normaloff>:/mainToolBar/more_colors.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_dispoSimulation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Disponibilité par simulation</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/mainToolBar/final.png</normaloff>:/mainToolBar/final.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_dispoGlobaleSimulation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Disponibilité globale par simulation</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/mainToolBar/initial.png</normaloff>:/mainToolBar/initial.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_calculEtats">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Calcul d'états</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/mainToolBar/window-new@2x.png</normaloff>:/mainToolBar/window-new@2x.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_calculPercolation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Calcul de percolation</string>
         </property>
         <property name="icon">
          <iconset resource="ressource.qrc">
           <normaloff>:/mainToolBar/document-open@2x.png</normaloff>:/mainToolBar/document-open@2x.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="SideBar" name="sideBar" native="true"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">QTabWidget{
	border: none;
}</string>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Objet</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Noeuds</string>
           </property>
           <item>
            <property name="text">
             <string>Noeud10</string>
            </property>
            <property name="text">
             <string>Noeud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Noeud25</string>
            </property>
            <property name="text">
             <string>Noeud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Noeud 100</string>
            </property>
            <property name="text">
             <string>Noeud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Noeud 1000</string>
            </property>
            <property name="text">
             <string>Noeud</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Liaisons</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <item>
            <property name="text">
             <string>Liaison 1</string>
            </property>
            <property name="text">
             <string>Liaison</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Liaison3</string>
            </property>
            <property name="text">
             <string>Liaison</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Liaison 50</string>
            </property>
            <property name="text">
             <string>Liaison</string>
            </property>
           </item>
          </item>
         </widget>
         <widget class="QTreeWidget" name="treeWidget_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string notr="true">Propriété</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Valeur</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Element</string>
           </property>
           <item>
            <property name="text">
             <string>Nom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Numéro</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disponibilité</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Couche</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Noeud</string>
           </property>
           <item>
            <property name="text">
             <string>Type</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Liaison</string>
           </property>
           <item>
            <property name="text">
             <string>Cout</string>
            </property>
           </item>
          </item>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
